src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.html

```diff
@@ -1,52 +1,55 @@
 <br>
-<ng-container>
-  <div class="card bg-primary">
-    <div class="card-body fill-plain">
-      <div class="form-group row">
-        <label class="col-sm-3 control-label bold-label text-right">Course ID:</label>
-        <div class="col-sm-3">
-          <input class="form-control" type="text" [(ngModel)]="newCourseId"
-                 ngbTooltip="Enter the identifier of the course, e.g.CS3215-2013Semester1."
-                 [maxlength]="40" placeholder="e.g. CS3215-2013Semester1"/>
+<div class="container">
+  <ng-container>
```

Comment:
Do we still need `<ng-container>` if you are wrapping the elements with an outer div now?

I've not pulled the code yet but does including `class="container"` help in certain ways?

Reply:
Yup, removed the unnecessary `<ng-container>`